### PR TITLE
Program for multiplication in 8085

### PR DESCRIPTION
**Pull request by gaurab-khanal**

### CODE REVIEW

File: programs/8085/q6.asm [hunk 1]
```diff
@@ -0,0 +1,15 @@
+; Write a program in 8085 to multiply two 8-bit data 05H and 08H and store the result at memory location 2500H.
+
+LXI H, 2500H  ; Load the memory location 2500H into the HL register pair.
+MVI A, 05H   ; Load the first 8-bit value (05H) into the accumulator (A).
+MOV B, A     ; Copy the value from A to B.
+MVI A, 00H   ; Initialize the accumulator (A) with 00H to store the result.
+
+LOOP:
+  ADD B      ; Add B to A.
+  DCR C      ; Decrement C (counter register).
```
Comment (aayush105) [inline programs/8085/q6.asm:10]: Counter C is not initialized, how this can be decremented?

Comment (gaurab-khanal) [reply programs/8085/q6.asm:10]: > Counter C is not initialized, how this can be decremented?
Let me fix this.

**Review by gaurovgiri — APPROVED** [review]
LGTM! Thank you @gaurab-khanal for your contribution. Hope to see more of it. 🥳

**Review by aayush105 — APPROVED** [review]
LGTM!

**Pull request merged**